demos/demo

```diff
@@ -23,7 +23,16 @@ echo_color() {
     echo "${cyan}$1${reset}"
 }
 
+check_command() {
+    if ! command -v "${1}" &> /dev/null
+    then
+        echo "${1} command could not be found. Install ${1} to continue."
+        exit 1
+    fi
+}
+
 init() {
+    check_command "sudo"
```

Comment:
Is it really worth it? just checking `sudo` but not checking the other commands?

Reply:
good catch, now that we have bash, let's use a for there to validate the commands. I just used sudo as I noticed e2e had no sudo and failed.